Runtime/Interaction/StaticMesh/HPUIStaticContinuousInteractable.cs

```diff
@@ -0,0 +1,28 @@
+using ubco.ovilab.HPUI.Interaction;
+using UnityEngine;
+
+namespace ubco.ovilab.HPUI.StaticMesh
+{
+    [RequireComponent(typeof(StaticMeshCollidersManager))]
+    public class HPUIStaticContinuousInteractable : HPUIBaseInteractable
+    {
+        [SerializeField] private SkinnedMeshRenderer staticHPUIMesh;
+        
+        [SerializeField] private int meshXRes;
```

Comment:
You may want to rename this to "meshXResolution" :)

Reply:
fixed 🫡